test/evp_kdf_test.c

```diff
@@ -885,11 +885,11 @@ static int test_kdf_scrypt(void)
 
     ret =
         TEST_ptr(kctx = get_kdfbyname(OSSL_KDF_NAME_SCRYPT))
-        && TEST_true(EVP_KDF_CTX_set_params(kctx, params))
-        /* failure test *//*
-        && TEST_int_le(EVP_KDF_derive(kctx, out, sizeof(out), NULL), 0)*/
+        && TEST_int_gt(EVP_KDF_CTX_set_params(kctx, params), 0)
+        /* failure test */
+        /* && TEST_int_le(EVP_KDF_derive(kctx, out, sizeof(out), NULL), 0) */
```

Comment:
Just remove this line.

Reply:
Thanks :-)

Reply:
It seems that the test fails again...

Reply:
Without an error message saying what's wrong.